source/PlayerInfo.cpp

```diff
@@ -2779,6 +2779,20 @@ void PlayerInfo::RegisterDerivedConditions()
 	flagshipModelProvider.SetHasFunction(flagshipModelFun);
 	flagshipModelProvider.SetGetFunction(flagshipModelFun);
 
+	auto &&outfitTagProvider = conditions.GetProviderPrefixed("outfit tag: ");
```

Comment:
I voiced [some concerns](https://github.com/endless-sky/endless-sky/pull/7482#discussion_r1014729762) on using `tag` in the keyword; for me tagging is something that happens in-game, when something happens. This is data that is always, by default, on the Outfits, so I would really like to see something else here.

Reply:
any suggestion?
attribute would be ambiguos

Reply:
how about:
subcategory
identifier

Reply:
Maybe property or characteristic?

Reply:
subcategory doesn't feel right, because this new keyword does not really identify categories (and Outfits are already part of a category). Identifier also doesn't feel right, because this new data does not uniquely identify a single outfit.

Reply:
Thought about it a bit more; we could also just use tags here (as is currently done) and have the in-game functionality use another keyword. (There is no absolute requirement to use the keyword `tag` there.)

Reply:
what do you mean with the "in-game functionality"?

Reply:
Sorry, I wasn't clear. What I was thinking about is [a proposal to allow tagging/marking ships during missions](https://github.com/endless-sky/endless-sky/issues/4930).
It's a feature which I really want to have as a content-creator, because it will make it possible to also do some story-lines around ships.

But it is not very relevant; the feature in 4930 can also use another keyword, and then we can just use the keyword tag here.

Reply:
If you are fine with it...
I mean I dont mind just keeping it how it is